docs/mainpage.dox.tmpl

```diff
@@ -317,4 +317,5 @@ System::out.println("testing...");
 
     @subsection jni_1_0 jni Module Version 1.0
     - initial public release
+    - fixed callFunction0 issue when running at 32-bit system
```

Comment:
please remove this release note - this is unreleased code - release notes are meant to document changes between releases